src/rendering/screenspacerenderable.cpp

```diff
@@ -510,9 +510,10 @@ float ScreenSpaceRenderable::scale() const {
     return _scale;
 }
 
-void ScreenSpaceRenderable::createShaders() {
-    ghoul::Dictionary dict = ghoul::Dictionary();
-
+void ScreenSpaceRenderable::createShaders(ghoul::Dictionary dict) {
+    if (!dict.hasKey("useAcceleratedRendering")) {
```

Comment:
Worth checking what happens if the dictionary value isn't set.  Would probably be worthwhile to have the shaderpreprocessor set a `0` on default and the skip this code entirely

Reply:
I agree - how do i do that?

Reply:
If you leave out this code, start OpenSpace and then look in the `cache` folder for the `ScreenSpaceProgram` folder (the name of the shader), it will have the complete compiled program in there.  If everything works as expected it should have replaced the `${useAcceleratedRendering}` with `0`